tests/backup/backup_partial_test.go

```diff
@@ -744,3 +744,250 @@ var _ = Describe("{PartialBackupSuccessWithPxAndKDMPVolumes}", func() {
+			srcClusterUid, err = Inst().Backup.GetClusterUID(ctx, BackupOrgID, SourceClusterName)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster uid", SourceClusterName))
+			log.Infof("Cluster [%s] uid: [%s]", SourceClusterName, srcClusterUid)
+			dstClusterStatus, err := Inst().Backup.GetClusterStatus(BackupOrgID, DestinationClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", DestinationClusterName))
+			dash.VerifyFatal(dstClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", DestinationClusterName))
+			destClusterUid, err = Inst().Backup.GetClusterUID(ctx, BackupOrgID, DestinationClusterName)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster uid", DestinationClusterName))
+			log.Infof("Cluster [%s] uid: [%s]", DestinationClusterName, destClusterUid)
+		})
+
+		Step("Create scheduled backups for backup validation", func() {
+			log.InfoD("Creating %v scheduled backups for namespaces [%s]", scheduledBackupCount, appNamespaces)
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			log.InfoD("Creating a new schedule policy for new apps for backup validation")
+			schedulePolicyName = fmt.Sprintf("backup-schedule-%v", RandomString(6))
+			schedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, int64(15), 5)
+			err = Inst().Backup.BackupSchedulePolicy(schedulePolicyName, uuid.New(), BackupOrgID, schedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of new schedule policy [%s]", schedulePolicyName))
+			schedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(BackupOrgID, ctx, schedulePolicyName)
+			log.FailOnError(err, "Fetching uid of schedule policy [%s]", schedulePolicyName)
+			var wg sync.WaitGroup
+			for i := 1; i <= scheduledBackupCount; i++ {
+				scheduleName := fmt.Sprintf("scheduled-backup-%v-%v", i, RandomString(6))
+				log.Infof("Creating scheduled backup [%s]", scheduleName)
+				wg.Add(1)
+				go func(i int) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					_, err = CreateScheduleBackupWithoutCheckWithVscMapping(scheduleName, SourceClusterName, backupLocationName, backupLocationUid, appNamespaces, labelSelectors, BackupOrgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx, map[string]string{}, true)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule [%s]", schedulePolicyName))
+					scheduledBackups = append(scheduledBackups, scheduleName)
+				}(i)
+				time.Sleep(timeBetweenScheduledBackups)
+			}
+			wg.Wait()
+		})
+
+		Step("Verify if the first scheduled backup is Success", func() {
+			log.InfoD("Verifying if the first scheduled backup is of Success status")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			var wg sync.WaitGroup
+			for _, schBkpName := range scheduledBackups {
```

Comment:
the term "scheduledBackups" is used in several places. To avoid confusion between the schedule name and the backup name, could we replace all instances of "scheduledBackups" with "scheduleNames"?

Reply:
Done